frontend/application-ie8.scss

```diff
@@ -1,3 +1,5 @@
+@import '../node_modules/accessible-autocomplete/dist/accessible-autocomplete.min';
```

Comment:
We were including this in our stylesheets with a concat during the CSS build process previously. I've moved here because I think it's clearer to build from a single source file. It also means we can remove `grunt-contrib-concat`.